.devcontainer/devcontainer.json

```diff
@@ -31,7 +31,7 @@
   // Configure tool-specific properties.
   "customizations": {
     "vscode": {
-      "extensions": ["vortizhe.simple-ruby-erb",
```

Comment:
I'm a bit suspicious that this extension (https://marketplace.visualstudio.com/items?itemName=vortizhe.simple-ruby-erb) could have been responsible for the frequent disappearance of syntax highlighting. I'll try to test this somehow to see if that's the case

Reply:
I agree that this is probably not causing the syntax highlighting disappearance issue now that you make that point about `.rb` files. I'm going to insert this extension back in. 

I think the syntax highlighting problem is coming from somewhere else and I can look into it in more detail on a separate PR. I am worried that could be a rabbit hole because I haven't found any issues anywhere reporting quite the issue we are having with the need for a page refresh. I have an open issue to continue to track this problem: https://github.com/appdev-lessons/codespaces-setup/issues/4